browser/js/app.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-window.app = angular.module('FullstackGeneratedApp', ['fsaPreBuilt', 'ui.router', 'ui.bootstrap', 'ngAnimate', 'gridstack-angular']);
+window.app = angular.module('FullstackGeneratedApp', ['fsaPreBuilt', 'ui.router', 'ui.bootstrap', 'ngAnimate', 'gridstack-angular', 'ui.bootstrap.modal']);
```

Comment:
probably don't need 'ui.bootstrap.modal' because that is included in 'ui.bootstrap'